specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json

```
@@ -1617,17 +1620,18 @@
       "properties": {
         "days": {
           "format": "int32",
-          "description": "The number of days to retain manifest before it expires.",
+          "description": "The number of days to retain an untagged manifest after which it gets purged.",
+          "default": 7.0,
```

If it's an in there is no decimal
```suggestion
          "default": 7,
```

---

Fixed.